assets/src/components/parts/janus-video/Video.tsx

```diff
@@ -454,7 +454,7 @@ const Video: React.FC<PartComponentProps<VideoModel>> = (props) => {
       height="100%"
       /* className={cssClass} */
       autoPlay={autoPlay}
-      loop={autoPlay}
+      loop={enableReplay}
```

Comment:
This is not related to this ticket. I saw this bug with  video component so fixed it here.